### PR TITLE
sharding a conversation code

### PR DESCRIPTION
**Pull request by aswinrajeevofficial**
This PR contains the code for sharding a conversation

### CODE REVIEW

File: howtos/how-to-shard-a-conversation/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  await Talk.ready;
+  const user = new Talk.User({
+    id: crypto.randomUUID(),
+    name: name,
+    email: email,
+    photoUrl: profilePictureURL,
+    role: "default"
+  });
+  talkSession = window.talkSession = new Talk.Session({
+    appId: "<YOUR_APP_ID>",
```
Comment (keerlu) [inline howtos/how-to-shard-a-conversation/index.js:69]: It threw me a bit having to paste in the app ID twice, can you extract them out into a variable?

File: howtos/how-to-shard-a-conversation/index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    catch: function () {
+                        return k && new k();
+                    },
+                    c: l,
+                },
+            };
+        })(window, document, []);
+    </script>
+    <script src="index.js" async defer></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"
```
Comment (keerlu) [inline howtos/how-to-shard-a-conversation/index.html:41]: Is Popper something you're using with Bootstrap? Would be good to mention it in the tutorial if it's required.

File: howtos/how-to-shard-a-conversation/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- [Node.js](https://nodejs.org/en) and [npm](https://www.npmjs.com/)
+
+## How to run the tutorial
+
+1. Clone or download the project.
+2. Replace `<YOUR_APP_ID>` in `index.js` and `<YOUR_APP_ID` and `<SECRET_KEY>` in `seed-users.js` with the values found in your [TalkJS dashboard](https://talkjs.com/dashboard/login).
+3. Go to your TalkJS dashboard, go the *Chat UI* tab and set the *team_chat* theme to the default role.
```
Comment (keerlu) [inline howtos/how-to-shard-a-conversation/README.md:14]:
```suggestion
3. Go to your TalkJS dashboard, go to the *Chat UI* tab and set the *team_chat* theme to the default role.
```

File: howtos/how-to-shard-a-conversation/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+4. Run `npm install` to install dependencies.
+5. Run `npm start` to start to create the sample users and conversations.
```
Comment (keerlu) [inline howtos/how-to-shard-a-conversation/README.md:16]:
```suggestion
5. Run `npm start` to create the sample users and conversations.
```

**Comment by aswinrajeevofficial** [comment]
Hi @keerlu, I've just pushed the changes as per your feedback. Popper is not required, so removed that. Thanks for your review.

**Comment by aswinrajeevofficial** [comment]
Hey @stevenwaterman, the changes look fantastic. I've also made changes to the article as per these changes.

**Pull request merged**